Plugins/Gerrit/FormGerritBase.cs

```diff
@@ -66,7 +66,15 @@ private Version GetGerritVersion()
         {
             RestRequest request = new RestRequest("/config/server/version");
             IRestResponse response = client.Execute(request);
-            return Version.Parse(response.Content.Replace(")]}'", "").Replace("\n", "").Replace("\"", "").Split('-')[0]);
+            if (response.IsSuccessful)
+            {
+                string versionText = response.Content.Replace(")]}'", "").Replace("\n", "").Replace("\"", "").Split('-')[0];
+                return Version.Parse(versionText);
```

Comment:
Wouldn't it be easier to replace this with a regex?
Something like \d{1,3}\.\d{1,3}(\.\d{1,3}){0,2}

Reply:
GitHub's eaten escaping, dot must be escaped too